### ISSUE TITLE
Feature request: ungrouped, titled windows in dock/panel

### ISSUE THREAD

**Issue by birgersp** [pop-os/cosmic-dock#190]
Hello,

I prefer to have my windows ungrouped and visible with window title, in the panel.

Currently, I achieve this with the (excellent) [dash-to-panel](https://extensions.gnome.org/extension/1160/dash-to-panel/) extension, with the "Ungroup applications" setting switched to off. Could you please add such a feature?

Here is a screenshot of my setup (notice the panel at the bottom of the image):

![image](https://github.com/pop-os/cosmic-dock/assets/5260237/475ec3f5-adc8-4f20-9452-c51c16cd7a56)

TLDR; feature request: display ungrouped windows with title, in the panel

**Comment by joewood9364** [comment]
Hi I think this is the old gnome dock issue tracker. The new COSMIC dock is at:
[https://github.com/pop-os/cosmic-applets/issues/297](https://github.com/pop-os/cosmic-applets/issues/297)

**Comment by birgersp** [comment]
> Hi I think this is the old gnome dock issue tracker. The new COSMIC dock is at: [pop-os/cosmic-applets#297](https://github.com/pop-os/cosmic-applets/issues/297)

oh ok, thanks!
